lib/app.tsx

```diff
@@ -177,7 +177,7 @@ class AppComponent extends Component<Props> {
       'note-info-open': showNoteInfo,
       'navigation-open': showNavigation,
       'is-electron': isElectron,
-      'is-macos': isMac,
+      'is-macos': isElectron && isMac,
```

Comment:
Why add the catch for isElectron here?

Reply:
From what I can tell the only place we are really using the `is-macos` class in styles is meant for the Mac Electron app around the size of the note toolbar and menu toolbar.